templates/data/kafka/index.html

```diff
@@ -16,225 +24,249 @@
-          {{ image(url="https://assets.ubuntu.com/v1/5dc854e4-Artboard%20154@4x.png",
+          ]
+        }
+      }
+    ]
+  ) -%}
+    {%- if slot == 'description' -%}
+    <p>
+      When you use Canonical's Kafka, you get the benefit of running on top of an operating system image that's optimised for every cloud. Run Kafka on Azure, AWS, on Kubernetes or your private data centre, with OpenStack or VMWare.
+    </p>
+    {%- endif -%}
+  {% endcall -%}
+
+  {{ vf_basic_section(
+    title={"text": "Easy to deploy and manage"},
+    items=[
+      {
+        "type": "image",
+        "item": {
+          "aspect_ratio": "16-9",
+          "attrs": image(url="https://assets.ubuntu.com/v1/5dc854e4-Artboard%20154@4x.png",
                     alt="",
                     width="422",
                     height="308",
                     hi_def=True,
                     loading="lazy",
-                    attrs={"class": "p-image-container__image"}) | safe
-          }}
-        </div>
-        <p>
-          Charmed Kafka is simply the best way to run Kafka, on the cloud or in your data centre. Runs directly on cloud infrastructure or on Kubernetes.
-        </p>
-        <ul class="p-list--divided">
-          <li class="p-list__item is-ticked">Automated security and configuration best practices built in</li>
-          <li class="p-list__item is-ticked">Integrated observability for monitoring, alerting and auditing</li>
-          <li class="p-list__item is-ticked">Multi-cloud support</li>
-        </ul>
-        <div class="p-cta-block">
-          <a href="/data/docs/kafka/iaas">Get started with Charmed Kafka&nbsp;&rsaquo;</a>
-        </div>
-      </div>
-    </div>
-  </section>
+                    output_mode="attrs")
+        }
+      },
+      {
+        "type": "description",
+        "item": {
+          "type": "text",
+          "content": "
+            Charmed Kafka is simply the best way to run Kafka, on the cloud or in your data centre. Runs directly on cloud infrastructure or on Kubernetes.
+          "
+        }
+      },
+      {
+        "type": "list",
+        "item": {
+          "list_items": [
+            {
+              "list_item_type": "tick",
+              "content": "Automated security and configuration best practices built in"
+            },
+            {
+              "list_item_type": "tick",
+              "content": "Integrated observability for monitoring, alerting and auditing"
+            },
+            {
+              "list_item_type": "tick",
+              "content": "Multi-cloud support"
+            }
+          ]
+        }
+      },
+      {
+        "type": "cta-block",
+        "padding": "shallow",
+        "item": {
+          "link": {
+            "content_html": "Get started with Charmed Kafka&nbsp;&rsaquo;",
+            "attrs": {
+              "href": "/data/docs/kafka/iaas"
+            }
+          }
+        }
+      }
```

Comment:
@muhammad-ali-pk The padding here didn't seem to do anything.
Couldn't also find it in the docs